src/utils.jl

```diff
@@ -209,11 +209,6 @@ maketuple(x::Real) = (x, x)
 maketuple(x::Tuple{T,S}) where {T,S} = x
 
 RecipesPipeline.unzip(v) = unzip(v)
-RecipesPipeline.unzip(points::AbstractVector{<:GeometryBasics.Point}) =
```

Comment:
On second thought, we can probably switch to `StaticArraysCore` to maintain this functionality with most of the load time benefits.

Reply:
Edit: this is not going to help with GeometryBasics.

Reply:
I think we can keep this behind a required block to not break to much things